Packs/ApiModules/Scripts/FireEyeApiModule/FireEyeApiModule.py

```diff
@@ -114,10 +114,11 @@ def _get_token(self) -> str:
 
     @logger
     def _generate_token(self) -> str:
-        resp = self._http_request(method='POST', url_suffix='auth/login', resp_type='response')
-        if resp.status_code != 200:
+        try:
+            resp = self._http_request(method='POST', url_suffix='auth/login', resp_type='response')
+        except DemistoException as er:
             raise DemistoException(
-                f'Token request failed with status code {resp.status_code}. message: {str(resp)}')
+                f'Token request failed. message: {str(er)}')
```

Comment:
cant we just cast the status code to str? anyway i think it would be useful to return it in the error message

Reply:
That is exactly the problem, once there's an error there is no resp object (so you can't do resp.status_code). In that case the err contains the code.